apps/meeting/.eslintrc.json

```diff
@@ -13,6 +12,9 @@
       "version": "detect"
     }
   },
+  "files": [
+    "src/*.(ts|tsx)"
```

Comment:
Now our eslint now only targets the react app source code. It uses `@typescript-eslint` and was also targeting `.js` file mistakenly.